docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -215,11 +215,24 @@ Using hyperkit requires having both `hyperkit` and `docker-machine-driver-hyperk
 
 ==== Installing hyperkit
 
-* If you have link:https://docs.docker.com/docker-for-mac/[Docker Desktop for macOS] installed, `hyperkit` is already installed. 
-* If you use link:https://brew.sh[Homebrew] you can install the latest version of `hyperkit`:
+* The following steps explain the installation of the *hyperkit* binary to the *_/usr/local/bin/_* directory:
```

Comment:
Just noticed this while adding the changes to the downstream docs. This isn't a list item, so we should remove the "* " before the rest of the text. Otherwise, everything looks great. :)

Reply:
Done

Reply:
Awesome, thank you! I already added these changes to the downstream documentation, so as far as I'm concerned, this can be merged.